CHANGES.md

```diff
@@ -7,6 +7,10 @@ Change Log
 * Added optional `index` parameter to `PrimitiveCollection.add`. [#8041](https://github.com/AnalyticalGraphicsInc/cesium/pull/8041)
 
 ##### Fixes :wrench:
+* Disable FXAA by default. To renable, `scene.postProcessStages.fxaa.enabled = true` [#7875](https://github.com/AnalyticalGraphicsInc/cesium/issues/7875)
+* Improve label legibility and overall visual quality by rendering at device native resolution regardless of the window.devicePixelRatio setting. [#7875](https://github.com/AnalyticalGraphicsInc/cesium/issues/7875)
```

Comment:
There should be instructions for going back to the old behavior by doing `viewer.resolutionScale = 1.0`.

Reply:
This depends on the decision for https://github.com/AnalyticalGraphicsInc/cesium/pull/8057#discussion_r314772801